src/main/cpp/chassis/states/DriveToFieldElement.cpp

```diff
@@ -154,7 +154,7 @@ void DriveToFieldElement::InitChassisMovement(ChassisMovement &chassisMovement)
     chassisMovement.pathnamegains = ChassisOptionEnums::PathGainsType::LONG;
     chassisMovement.chassisSpeeds.omega = units::angular_velocity::radians_per_second_t(0);
     chassisMovement.checkTipping = false;
-    chassisMovement.tippingTolerance = units::angle::degree_t(5.0);
+    chassisMovement.tippingTolerance = units::angle::degree_t(10.0);
```

Comment:
start this at 15 deg

Reply:
You should remove this from drive to field element so you don't have 2 different values to tune/change. This way we keep whatever is in holonomic drive